public/app/features/admin/state/reducers.ts

```diff
@@ -129,7 +137,32 @@ export const ldapUserReducer = reducerFactory(initialLdapUserState)
   })
   .create();
 
+export const userAdminReducer = reducerFactory(initialUserAdminState)
```

Comment:
Reducers are very easily tested with [reducerTester](https://github.com/grafana/grafana/blob/master/contribute/style-guides/redux.md#reducertester) so you should add tests as well.